app/components/FileLoader.js

```diff
@@ -302,6 +318,28 @@ export default class FileLoader extends Component {
       this
     );
 
+    const bufferMoreFiles = () => {
+      const fileBufferSize = 10;
```

Comment:
Suggestion: Pull this out to the top of the file as a const, and give a better name like `GAME_BATCH_SIZE`